### PR TITLE
git cheat sheet

### PR DESCRIPTION
**Pull request by zekiel**
What does your Instant Answer do?
Shows a cheat sheet for git commands.

What problem does your Instant Answer solve (Why is it better than organic links)?
Doesn't require clicking. 

What is the data source for your Instant Answer? (Provide a link if possible)
Github's own training docs. 

Why did you choose this data source?
They happen to be an authority on the matter. 

Are there any other alternative (better) data sources?
No. If there are, they can be added on. 

What are some example queries that trigger this Instant Answer?
git cheat sheet

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)
Anyone who uses github! 

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?
Nope

Which existing Instant Answers will this one supercede/overlap with?
None

Are you having any problems? Do you need our help with anything?
The JSON parsing doesn't work perfectly with my edge-case cheat sheet. 

Where did you hear about DuckDuckHack? (For first time contributors)
I helped create it. 
screen shot 2015-06-01 at 6 49 16 pm

More work will need to be done before this is polished off! 

Checklist

- [x] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [x] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

100%
(1:0)

### CODE REVIEW

File: share/goodie/cheat_sheets/git.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
+{
+    "id": "git_cheat_sheet",
+    "name": "Git",
+    "description": "Git help",
+    "metadata": {
+        "sourceName": "Github.com",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/git.json]: I would switch this to `GitHub` (we don't typically mention the domain)

**Comment by moollaza** [comment]
Other than the small update to the source, this LGTM :+1:

**Comment by iambibhas** [comment]
Just curious, why aren't some keys code formatted? e.g. `git config --global`

**Comment by javathunderman** [comment]
@zekiel This looks awesome.

**Comment by moollaza** [comment]
@iambibhas that was a mistake on the parser's part -- all of them will be wrapped in code blocks when it goes live. A PR was just merged into CheatSheets.js, so if you run it locally now, they'll all be correct.

**Comment by zekiel** [comment]
@moollaza updated source name. Should be good to merge.

**Comment by moollaza** [comment]
LGTM :+1:

**Pull request merged**